vet.sh

```diff
@@ -122,7 +122,7 @@ done
 # TODO(dfawley): don't use deprecated functions in examples or first-party
 # plugins.
 SC_OUT="$(mktemp)"
-staticcheck -go 1.9 -checks 'inherit,-ST1015' ./... > "${SC_OUT}" || true
+staticcheck -go 1.19 -checks 'inherit,-ST1015,-ST1019,-SA1019' ./... > "${SC_OUT}" || true
```

Comment:
Should we add a TODO saying we should take care of the double imports though our vet script?

Reply:
Yeah, good idea; we can programmatically ignore instead of totally disabling.